Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -317,13 +317,21 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
             sys.stderr.write('Checks have failed, aborting pull request.\n')
             return 1
 
-        issue = None
+        issues = []
         commits = list(repository.commits(begin=dict(hash=branch_point.hash), end=dict(branch=repository.branch)))
         for line in commits[0].message.split() if commits[0] and commits[0].message else []:
             issue = Tracker.from_string(line)
             if issue:
+                issues.append(issue)
+            if not line:
                 break
```

Comment:
Seems like this will reject a commit message like:

```
Subject line

https://bugs.webkit.org/nnnnnn
rdar://nnnnn
```

I know our template doesn't add an empty line here, but I don't think it's unreasonable for contributors to do so, especially because some text editors treat the second line of a commit message separately from the main body.

What if the logic was: Skip the first line, then collect issues until the first nonempty, non-issue line?

Reply:
We need to consider the first line (the Safari team, notably, puts their bug numbers on the first line), that being said, I think your point about the 1st empty line is a good one, I'll change this code to break out on the 2nd empty line.